LayoutTests/http/wpt/webauthn/public-key-credential-create-failure-hid.https-expected.txt

```diff
@@ -12,4 +12,5 @@ PASS PublicKeyCredential's [[create]] with unsupported options in a mock hid aut
 PASS PublicKeyCredential's [[create]] with mixed options in a mock hid authenticator.
 PASS PublicKeyCredential's [[create]] with mixed options in a mock hid authenticator. 2
 PASS PublicKeyCredential's [[create]] with InvalidStateError in a mock hid authenticator.
+PASS PublicKeyCredential's [[create]] with full key store.
```

Comment:
Can you work to move these tests into WPT proper? These seem like things that should be tested on the official WPT suite.